resources/agent.rb

```diff
@@ -37,7 +37,31 @@
                                  }
 action :install do
   # Linux installation - source package
```

Comment:
This needs to be removed or updated, if updated, commented should be added above the else indicating the Linux block

Reply:
Yeah, my bad.